persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -731,6 +738,7 @@ mayDefault def = case def of
     Just d -> " DEFAULT " <> d
 
 type SafeToRemove = Bool
+type SchemaEntityName = (Text, EntityNameDB)
```

Comment:
I'd prefer to see a `newtype` or a `data` with named fields to make this more explicit